kymatio/scattering1d/frontend/tensorflow_frontend.py

```diff
@@ -15,7 +15,27 @@ def __init__(self, J, shape, Q=1, T=None, max_order=2, oversampling=0,
         ScatteringBase1D.build(self)
         ScatteringBase1D.create_filters(self)
 
+
 ScatteringTensorFlow1D._document()
 
 
-__all__ = ['ScatteringTensorFlow1D']
+class TimeFrequencyScatteringTensorflow(
+        ScatteringTensorFlow, TimeFrequencyScatteringBase):
+    def __init__(self, *, J, J_fr, shape, Q, T=None, oversampling=0, Q_fr=1,
+            F=None, oversampling_fr=0, out_type='array', backend='tensorflow', name="TimeFrequencyScatteringScattering"):
```

Comment:
try unifying between single quotes `out_type='array'` and double quotes `name="TimeFrequencyScattering"`

also ... `name="TimeFrequencyScattering"`, not `"TimeFrequencyScatteringScattering"`